test/api/test.api

```diff
@@ -1,3 +1,18 @@
+public final class com/juul/tuulbox/test/AssertKt {
+	public static final fun assertContains (Ljava/lang/Iterable;Ljava/lang/Object;)V
+	public static final fun assertContains (Ljava/util/Collection;Ljava/lang/Object;)V
+	public static final fun assertContains (Lkotlin/ranges/ClosedRange;Ljava/lang/Comparable;)V
+	public static final fun assertContains (Lkotlin/sequences/Sequence;Ljava/lang/Object;)V
+	public static final fun assertContains ([Ljava/lang/Object;Ljava/lang/Object;)V
+	public static final fun assertSimilar (DDD)V
+	public static final fun assertSimilar (FFF)V
+	public static final fun assertSimilar (III)V
+	public static final fun assertSimilar (JJJ)V
+	public static final fun assertSimilar-2yBKrUY (Lkotlinx/datetime/LocalDateTime;DLkotlinx/datetime/LocalDateTime;Lkotlinx/datetime/TimeZone;)V
+	public static synthetic fun assertSimilar-2yBKrUY$default (Lkotlinx/datetime/LocalDateTime;DLkotlinx/datetime/LocalDateTime;Lkotlinx/datetime/TimeZone;ILjava/lang/Object;)V
+	public static final fun assertSimilar-hN_NL7I (Lkotlinx/datetime/Instant;DLkotlinx/datetime/Instant;)V
```

Comment:
Are you aware of a ware to prevent name mangling on our public API? Or is this by design, as they're `internal` so shouldn't be used?

Reply:
These functions are properly `public`, not `@PublishedApi internal`. The mangling here is because the function accepts a `kotlin.time.Duration` argument (which is an `inline class` wrapper around a `Double`).